### PR TITLE
Introducing new western sky culture sternenkarten.com

### PR DESCRIPTION
**Pull request by seescho**
### Description
Introduce the new western sky culture sternenkarten.com

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] This change requires a documentation update

### How Has This Been Tested?
Compiling, installing and testing on openSUSE Tumbleweed.
**Test Configuration**:
* Operating system: openSUSE Tumbleweed
* Graphics Card: Intel-chipset graphics

## Checklist:

- [x] My code follows the [code style](http://stellarium.org/doc/head/codingStyle.html) of this project.
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by alex-w — CHANGES_REQUESTED** [review]
I see `star_names.fab` has references, but file `references.fab` is missing

File: skycultures/western_sternenkarten/info.ini [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[info]
+name = Western (Sternenkarten)
+author = Edgar Scholz
+descriptionSource = https://sternenkarten.com/die-sternbilder/
+boundaries = iau
```
Comment (alex-w) [inline skycultures/western_sternenkarten/info.ini:5]: Please add license info here, for example: `license = CC BY-ND 4.0 International`

**Comment by alex-w** [comment]
Please check items from QA checklist also

**Review by alex-w — APPROVED** [review]

**Comment by seescho** [comment]
> Please check items from QA checklist also

done

**Review by alex-w — APPROVED** [review]
Thank you!

**Pull request merged**